test/fixtures/internal-modules/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('internal/freelist');
```

Comment:
Shouldn't this file be called freelist.js or something?  Avoids unnecessary churn when another module is added.